core/src/main/resources/db/changelog/changesets/2023_08_25_admin_task.yaml

```diff
@@ -0,0 +1,43 @@
+databaseChangeLog:
+  - changeSet:
+      id: "adminTask"
+      author: dbush
+      changes:
+        - createTable:
+            tableName: admin_task
+            columns:
+              - column: { name: id, type: uuid, defaultValueComputed: gen_random_uuid(), constraints: { nullable: false, primaryKey: true } }
+              - column: { name: created_at, type: datetime, constraints: { nullable: false } }
+              - column: { name: last_updated_at, type: datetime, constraints: { nullable: false } }
+              - column: { name: completed_at, type: datetime }
+              - column: { name: status, type: text, constraints: { nullable: false } }
+              - column: { name: disposition_note, type: text }
+              - column: { name: description, type: text }
+              - column: { name: study_environment_id, type: uuid, constraints:
+                { nullable: false, foreignKeyName: fk_admin_task_study_env, references: study_environment(id) }
+              }
+              - column: { name: participant_note_id, type: uuid, constraints:
```

Comment:
yup, you're right -- we do want to be able to attach tasks to lots of different things.  `nullable: true` is the default, so this column will be nullable since we don't explicitly say `nullable: false` like we do for some of  the above columns